realm/realm-library/build.gradle

```diff
@@ -46,6 +46,7 @@ repositories {
 
 dependencies {
     provided 'io.reactivex:rxjava:1.1.0'
+    compile "com.android.support:recyclerview-v7:23.1.1"
```

Comment:
Please check [v7-recycler-method-count](http://www.methodscount.com/?lib=com.android.support%3Arecyclerview-v7%3A23.1.1), it gives around 11k, so pretty big. By the way I think proguard will shrink method count, please correct me if I am wrong.